src/hotspot/share/gc/g1/vmStructs_g1.hpp

```diff
@@ -53,7 +53,6 @@
   nonstatic_field(G1HeapRegionTable, _shift_by,         uint)                 \
                                                                               \
   nonstatic_field(HeapRegionManager, _regions,          G1HeapRegionTable)    \
-  nonstatic_field(HeapRegionManager, _num_committed,    uint)                 \
```

Comment:
If this field is remaining in hotspot, it should also remain in vmStructs. Although SA does not need to explicitly know about this field to function properly, SA can be used to dump hotspot objects, and can only dump objects whose types are in vmStructs, and for those types can only display the fields that are also in vmStructs.

Reply:
The field will remain for now, but the plan is to refactor it into a new class. The new class is not needed by the SA and I would really like to avoid having to add unused and untested code to the SA. To simplify this coming change I want to remove this from the SA straight away. 

Even without this coming change I would vote for removing this type of unused code from the SA. I don't think being able to dump the value is worth enough compared to the maintenance cost of keeping it in.

Reply:
Ok. If the field is going away eventually then you can remove it now.

Reply:
Awesome, thanks for reviewing.